test/testConfigure.js

```diff
@@ -0,0 +1,575 @@
+'use strict';
+
+// Import this first so we can use it to wrap other modules we import.
+var promisify = require("promisify-node");
+
+var assert = require('assert');
+var expect = require('chai').expect;
+var childProcess = require('child_process');
+var nock = require('nock');
+var readYaml = require('read-yaml');
+var temp = promisify(require('temp').track());
+
+var configure = require('../lib/configure');
+
+function await(data) {
+  return new Promise(function(resolve, reject) {
+    var write = process.stdout.write;
+    var output = '';
+    process.stdout.write = function(chunk, encoding, fd) {
+      write.apply(process.stdout, arguments);
+      output += chunk;
+      if (output.indexOf(data) !== -1) {
+        process.stdout.write = write;
+        resolve();
+      }
+    };
+  });
+}
+
+function emit(data) {
+  process.stdin.emit('data', data);
+}
+
+function cancel() {
+  emit(String.fromCharCode(3 /* Ctrl-C */));
+}
+
+describe('Configure', function() {
+  var slug = 'mozilla/oghliner', user = 'mozilla', repo = 'oghliner';
+  // Some test functions cause Oghliner to wait five seconds before repeating
+  // a request to the Travis API, so we need to increase the timeout
+  // to accommodate that delay.  I'd rather increase for just those specific
+  // tests, but it isn't clear how to do that for tests that return a promise.
+  this.timeout(10000);
```

Comment:
> Can't you call this.timeout inside a it to achieve this?

Ah, indeed, and the [mocha docs describe this](http://mochajs.org/#timeouts). I dunno why I missed it when I read them originally. Fixed in 6b1ec14.